packages/react/src/components/accordion/Accordion.tsx

```diff
@@ -56,6 +68,17 @@ export type CommonAccordionProps = React.PropsWithChildren<{
    * @default false
    */
   initiallyOpen?: boolean;
+  /**
+   * The language of the component. It affects which language is used for the close button text.
+   *
+   * @default "fi"
+   */
+  language?: Language;
```

Comment:
Would it be possible to require language property if closeButton exists?
I guess it shows an empty button or something if it is not provided

Reply:
Yes, I guess it does.